p2p/libp2p/connectionMonitor/libp2pConnectionMonitor2.go

```diff
@@ -0,0 +1,124 @@
+package connectionMonitor
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/multiformats/go-multiaddr"
+)
+
+type libp2pConnectionMonitor2 struct {
+	chDoReconnect              chan struct{}
+	reconnecter                p2p.Reconnecter
+	thresholdMinConnectedPeers int
+	sharder                    Sharder
+}
+
+// NewLibp2pConnectionMonitor2 creates a new connection monitor (version 2 that is more streamlined and does not care
+//about pausing and resuming the discovery process)
+func NewLibp2pConnectionMonitor2(reconnecter p2p.Reconnecter, thresholdMinConnectedPeers int) (*libp2pConnectionMonitor2, error) {
```

Comment:
I do not like "2" in the name, make a better name for it.

Reply:
libp2pConnectionMonitorSimple will be. Thanks.